mlsdbtool/tool_wrapper.sh

```diff
@@ -0,0 +1,278 @@
+MCC[308]="PM"
+MCC[310]="BM GU MP PR US VI"
+MCC[311]="AS GU US VI"
+MCC[312]="US"
+MCC[313]="PR US"
+MCC[314]="US"
+MCC[316]="US"
+MCC[330]="PR"
+MCC[334]="MX"
+MCC[338]="BM JM KY LC TC"
+MCC[340]="BL GF"
+MCC[342]="BB"
+MCC[344]="AG"
+MCC[346]="KY"
+MCC[348]="VG"
+MCC[350]="BM"
+MCC[352]="GD"
+MCC[354]="MS"
+MCC[356]="KN"
+MCC[358]="LC"
+MCC[360]="VC"
+MCC[362]="BQ"
+MCC[363]="AW"
+MCC[364]="BS"
+MCC[365]="AI"
+MCC[366]="DM"
+MCC[368]="CU"
+MCC[370]="DO"
+MCC[372]="HT"
+MCC[374]="TT"
+MCC[376]="TC"
+MCC[400]="AZ"
+MCC[401]="KZ"
+MCC[402]="BT"
+MCC[404]="IN"
+MCC[405]="IN"
+MCC[410]="PK"
+MCC[412]="AF"
+MCC[413]="LK"
+MCC[414]="MM"
+MCC[415]="LB"
+MCC[416]="JO"
+MCC[417]="SY"
+MCC[418]="IQ"
+MCC[419]="KW"
+MCC[420]="SA"
+MCC[421]="YE"
+MCC[422]="OM"
+MCC[424]="AE"
+MCC[425]="IL PS"
+MCC[426]="BH"
+MCC[427]="QA"
+MCC[428]="MN"
+MCC[429]="NP"
+MCC[432]="IR"
+MCC[434]="UZ"
+MCC[436]="TJ"
+MCC[437]="KG"
+MCC[438]="TM"
+MCC[440]="JP"
+MCC[441]="JP"
+MCC[450]="KR"
+MCC[452]="VN"
+MCC[454]="HK"
+MCC[455]="MO"
+MCC[456]="KH"
+MCC[457]="LA"
+MCC[460]="CN"
+MCC[466]="TW"
+MCC[467]="KP"
+MCC[470]="BD"
+MCC[472]="MV"
+MCC[502]="MY"
+MCC[505]="AU NF"
+MCC[510]="ID"
+MCC[514]="TL"
+MCC[515]="PH"
+MCC[520]="TH"
+MCC[525]="SG"
+MCC[528]="BN"
+MCC[530]="NZ"
+MCC[536]="NR"
+MCC[537]="PG"
+MCC[539]="TO"
+MCC[540]="SB"
+MCC[541]="VU"
+MCC[542]="FJ"
+MCC[543]="WF"
+MCC[544]="AS"
+MCC[545]="KI"
+MCC[546]="NC"
+MCC[547]="PF"
+MCC[548]="CK"
+MCC[549]="WS"
+MCC[550]="FM"
+MCC[551]="MH"
+MCC[552]="PW"
+MCC[553]="TV"
+MCC[554]="TK"
+MCC[555]="NU"
+MCC[602]="EG"
+MCC[603]="DZ"
+MCC[604]="MA"
+MCC[605]="TN"
+MCC[606]="LY"
+MCC[607]="GM"
+MCC[608]="SN"
+MCC[609]="MR"
+MCC[610]="ML"
+MCC[611]="GN"
+MCC[612]="CI"
+MCC[613]="BF"
+MCC[614]="NE"
+MCC[615]="TG"
+MCC[616]="BJ"
+MCC[617]="MU"
+MCC[618]="LR"
+MCC[619]="SL"
+MCC[620]="GH"
+MCC[621]="NG"
+MCC[622]="TD"
+MCC[623]="CF"
+MCC[624]="CM"
+MCC[625]="CV"
+MCC[626]="ST"
+MCC[627]="GQ"
+MCC[628]="GA"
+MCC[629]="CG"
+MCC[630]="CD"
+MCC[631]="AO"
+MCC[632]="GW"
+MCC[633]="SC"
+MCC[634]="SD"
+MCC[635]="RW"
+MCC[636]="ET"
+MCC[637]="SO"
+MCC[638]="DJ"
+MCC[639]="KE"
+MCC[640]="TZ"
+MCC[641]="UG"
+MCC[642]="BI"
+MCC[643]="MZ"
+MCC[645]="ZM"
+MCC[646]="MG"
+MCC[647]="YT"
+MCC[648]="ZW"
+MCC[649]="NA"
+MCC[650]="MW"
+MCC[651]="LS"
+MCC[652]="BW"
+MCC[653]="SZ"
+MCC[654]="KM"
+MCC[655]="ZA"
+MCC[657]="ER"
+MCC[658]="SH"
+MCC[659]="SS"
+MCC[702]="BZ"
+MCC[704]="GT"
+MCC[706]="SV"
+MCC[708]="HN"
+MCC[710]="NI"
+MCC[712]="CR"
+MCC[714]="PA"
+MCC[716]="PE"
+MCC[722]="AR"
+MCC[724]="BR"
+MCC[730]="CL"
+MCC[732]="CO"
+MCC[734]="VE"
+MCC[736]="BO"
+MCC[738]="GY"
+MCC[740]="EC"
+MCC[742]="GF"
+MCC[744]="PY"
+MCC[746]="SR"
+MCC[748]="UY"
+MCC[750]="FK"
+
+infile=$1
+country=`echo $2 | tr [:lower:] [:upper:]`
+if [ -z $country ] ; then
+    echo "Usage: $0 [mls_data_file] [country_code]" >&2
+    exit 1
+fi
+
+if [ ! -e $infile ] ; then
+    echo "ERROR: Can't find infile $infile" >&2
+    exit 1
+fi
+
+if [ ! -x "./geoclue-mlsdb-tool" ] ; then
```

Comment:
To get used e.g. with the mlsdb-data repository, suppose this becomes now the mlsdb-tool, and the binary its internal helper?